tests/specs/analog/downlinkCommands.spec.js

```diff
@@ -9,7 +9,7 @@ test.describe('analog downlink commands - parse hex dumps', () => {
         await page.goto(baseURL);
     });
 
-    test.afterEach(async ({page}) => page.getByLabel(fixture.logs.buttons.deleteLogs).click());
+    test.afterEach(( {page} ) => page.getByLabel(fixture.logs.buttons.deleteLogs).click());
```

Comment:
```suggestion
    test.afterEach(({page}) => page.getByLabel(fixture.logs.buttons.deleteLogs).click());
```

spaces here were required because of `async`, not anymore

Reply:
ok, removed